.circleci/config.base.yml

```diff
@@ -636,19 +637,21 @@ workflows:
           requires:
             - build
       - build_pkg_binaries:
-          context: amplify-cli-ecr
+          context: amplify-ecr-image-pull
           requires:
             - publish_to_local_registry
       - amplify_sudo_install_test:
-          context: amplify-cli-ecr
+          context: amplify-ecr-image-pull
           requires:
             - publish_to_local_registry
           filters:
             branches:
               only:
                 - master
       - amplify_e2e_tests:
-          context: amplify-cli-ecr
+          context:
```

Comment:
I think we should have another context for the test account tokens and don't rely on the global env vars at all